asip_v2/utility.py

```diff
@@ -1,76 +1,82 @@
 import argparse
-import datetime
 import os
 import random
-from os.path import basename
+from os.path import dirname, join
 
 import numpy as np
 import tensorflow as tf
 from tensorflow.keras import layers
 from tensorflow.keras.models import Model
 
 from archive import Archive
-from data_generator import DataGenerator
 
+def parse_common_args():
+    """Instantiates and creates common arguments of parser which works with 'argparse' of python."""
+    parser = argparse.ArgumentParser(description='Process the arguments of script')
+    parser.add_argument()
```

Comment:
Is that used? I couldnt find...

Reply:
Sorry, I forgot to remove that during development.

Reply:
Done in the next commit